app/helpers/reactjs_helper.rb

```
@@ -0,0 +1,7 @@
+module ReactjsHelper
+  def mount_react_component(name, selector, data)
+    javascript_tag defer: 'defer' do
+      "$(tfm.reactMounter.mount('#{name}', '#{selector}', #{data}));".html_safe
```

not using html_safe breaks the app

---

Have you tried safe_join as houndci says?

---

@dLobatog yes, I tried the hound suggestion. It breaks the application. @tbrisker said to ignore the error since html_safe is widely used throughout the application.